docs/examples/000/1. hello-world/readme.md

```diff
@@ -0,0 +1,96 @@
+# Hello-world sample
+This bicep sample is the canonical hello world.
+
+## Code description
+This bicep file takes a `yourName` parameter and adds that to a `hello` variable and
+returns the concatenated string as an ARM output.
+
+## Deploy process
+The command to deploy this bicep file is:
+
+### PowerShell
+``` PowerShell
+New-AzResourceGroupDeployment  -ResourceGroupName myRG -TemplateFile .\main.bicep
+```
+
+### Azure CLI
+``` Bash
+az deployment group create --resource-group myRG --template-file ./main.bicep
+```
+
+
+## Output

```

Comment:
What are your thoughts on only requiring the "Output" section of the readme to have the output names & values, not the full body of the output returned by PS/CLI? That would simplify this readme. So in this case we would have something like:

## Output

```
helloWorld: Hello World! - Hi Chris
```

Reply:
My thoughts on the Hello World! was to have full outputs, both PS and CLI.  I certainly think the other samples in here would be better served as just the actual outputs as you've indicated above.